0.14.0 fails to build from crates.io and cargo install

`cargo +nightly install papyrus` and `cargo +nightly install --path .` fail to build with the following error:

```
error[E0433]: failed to resolve: could not find `Backtrace` in `backtrace`
   --> src/run/mod.rs:152:36
    |
152 |         let backtrace = backtrace::Backtrace::new();
    |                                    ^^^^^^^^^ could not find `Backtrace` in `backtrace`

error[E0412]: cannot find type `Backtrace` in crate `backtrace`
   --> src/run/mod.rs:230:27
    |
230 |     backtrace: backtrace::Backtrace,
    |                           ^^^^^^^^^
    | 
   ::: /home/mike/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.42/src/print.rs:15:1
    |
15  | pub struct BacktraceFmt<'a, 'b> {
    | ------------------------------- similarly named struct `BacktraceFmt` defined here
    |
help: a struct with a similar name exists
    |
230 |     backtrace: backtrace::BacktraceFmt,
    |                           ^^^^^^^^^^^^
help: possible candidate is found in another module, you can import it into scope
    |
2   | use std::backtrace::Backtrace;
    |
```

Looks like the backtrace crate has been moved into the std library for the nightly toolchain: https://doc.rust-lang.org/std/backtrace/index.html

--- a/.gitignore
+++ b/.gitignore
@@ -2,5 +2,4 @@
 **/*.rs.bk
 PUSHING_PROCEDURE.md
 Cargo.lock
-test-script.sh
 codecov-*.sh

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,7 +20,7 @@ cmdtree =	    { version = "0.10",	default-features = false }
 kserd =		    { version = "0.3",	default-features = false,   optional = false,	features = [ "format" ] }
 
 # crates.io
-backtrace =	    { version = "0.3",	default-features = false }
+backtrace =	    { version = "0.3",	default-features = false,   optional = false,	features = [ "std" ] }
 colored =	    { version = "1",	default-features = false }
 crossbeam-channel = { version = "0.4",	default-features = false }
 crossterm =	    { version = "0.14",	default-features = false,   optional = true }

--- a/test-script.sh
+++ b/test-script.sh
@@ -1,0 +1,32 @@
+set -e
+
+# Build external_crate
+cd test-resources/external_crate
+cargo clean
+cargo build
+cd ../..
+# Build external_kserd
+cd test-resources/external_kserd
+cargo clean
+cargo build
+cd ../..
+
+# Update docs
+cargo modoc
+cargo +stable fmt
+
+# Check formatting
+cargo +stable fmt -- --check
+
+# Check clippy
+# cargo clippy --all-features -- -D warnings 
+
+# Run tests; includes interface testing
+cargo test --all-features -- --test-threads=1
+
+# Run cargo docs and ensure the linking is working
+cargo doc
+
+# To build docs, need clean target directory
+# mdbook build docs
+# mdbook test -L target/debug,target/debug/deps docs

